interpretation/deepseismic_interpretation/azureml_pipelines/README.md

```diff
@@ -0,0 +1,175 @@
+# Integrating with AzureML
```

Comment:
![image](https://user-images.githubusercontent.com/8856246/74266178-b5710d80-4cb8-11ea-986c-a1f806cf9903.png)

Reply:
@annazietlow

Reply:
Adding a pointer in the main README.md to the AML specific README.md in this commit https://github.com/microsoft/seismic-deeplearning/pull/195/commits/342f9be730bdd5831cb1b16ecfb8a105675f3fde. Would you like to see additional guidance on the main page?

Reply:
Looks good